climada/entity/exposures/crop_production.py

```diff
@@ -20,6 +20,9 @@
 import math
 import copy
 from pathlib import Path
+import os
+from os import listdir
+from os.path import isfile, join
```

Comment:
As a new coding convention in CLIMADA, `os.path` should not be used, but be replaced by functionality provided by `pathlib`.